updatehub/src/object/installer/copy.rs

```diff
@@ -71,16 +71,13 @@ impl Installer for objects::Copy {
                         .open(&dest)?,
                 );
 
-                let metadata = dest.metadata()?;
-                let orig_mode = metadata.permissions().mode();
-                metadata.permissions().set_mode(0o100_666);
```

Comment:
I believe this cannot be dropped. One possible corner case we need to cover is:

1. file has permission 444
1. we update file
1. file still has permission 444